docs/diagrams/FindSequenceDiagram.puml

```diff
@@ -0,0 +1,63 @@
+@startuml
+!include style.puml
+'https://plantuml.com/sequence-diagram
+
+box Logic LOGIC_COLOR_T1
+participant ":LogicManager" as LogicManager LOGIC_COLOR
+participant ":SocialiteParser" as SocialiteParser LOGIC_COLOR
+participant ":FindCommandParser" as FindCommandParser LOGIC_COLOR
+participant "f:FindCommand" as FindCommand LOGIC_COLOR
+end box
+
+box Model MODEL_COLOR_T1
+participant ":Model" as Model MODEL_COLOR
+participant "p:ContainsKeywordsPredicate" as ContainsKeywordsPredicate MODEL_COLOR
```

Comment:
Hmm is it possible to shift `p:ContainsKeywordsPredicate` ahead of `:Model` since it's activated earlier than `:Model`? If possible, we should also close the gap between the two boxes `Logic` and `Model`